.github/workflows/tests.yml

```diff
@@ -106,10 +106,10 @@ jobs:
         run: cargo fmt --all -- --check
 
       - name: Clippy
-        run: cargo clippy -- -D warnings
+        run: cargo clippy
```

Comment:
Denying warnings was a bit aggressive for CI since it's running a matrix of stable, beta and nightly, where rules can be present on one toolchain but not on another.